docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -89,11 +89,11 @@ Make sure all 5.x indices have been deleted before upgrading to {version}. {es}
 {version} will fail to start if any 5.x indices are present.
 
 If you are running a version prior to 6.0,
-https://www.elastic.co/guide/en/elastic-stack/6.7/upgrading-elastic-stack.html[upgrade to 6.7]
+https://www.elastic.co/guide/en/elastic-stack/6.8/upgrading-elastic-stack.html[upgrade to 6.8]
 and reindex your old indices or bring up a new {version} cluster and
 {ref}/reindex-upgrade-remote.html[reindex from remote].
 
-The recommended path is to upgrade to 6.7 before upgrading to {version}. This
+The recommended path is to upgrade to 6.8 before upgrading to {version}. This
```

Comment:
Just wonder if this is strong enough language around 'we recommend you upgrade from 6.8 to 7.1' due to changed security licensing. (I admit, I'm having trouble wrapping my brain around the recommended upgrade path.)

Reply:
I don't think the changes to license changes the recommendations here. Here it is recommended so you can do a rolling restart (as opposed to a full cluster restart) which was true before and after the license change.

Reply:
The recommended upgrade path is through the last minor of the previous major to the most recent minor. So now 6.8 -> 7.1.